src/sparsezoo/model/utils.py

```diff
@@ -73,11 +119,20 @@ def load_files_from_directory(directory_path: str) -> List[Dict[str, Any]]:
     return files
 
 
+def _get_compressed_size(request_json: List[Dict[str, Any]]) -> int:
```

Comment:
Perhaps this function should return `None` if this file does not exist? This would also help support loading from a directory without this file.

Reply:
+1

Reply:
Accepted!